src/main/java/mate/controller/IndexController.java

```diff
@@ -5,11 +5,16 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
 
 public class IndexController extends HttpServlet {
+    private HttpSession session;
+
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
+        session = req.getSession();
+        req.setAttribute("driver",session.getAttribute("name"));
```

Comment:
```suggestion
        req.setAttribute("driver", session.getAttribute("name"));
```

I think it's better to make "driver" as constant field